### DISCUSSION TITLE
SSL Chown cmd equivalent in Windows

### DISCUSSION THREAD

**Discussion by treycalvertKM** [apache/couchdb#3035] [Q&A] [answered]
I'm currently trying to make couchdb https instead of http I have the both crts the last step before enabling the ssl is this command: chown couchdb server.crt server.key unfortunately I'm on a windows machine and i cant seem to find an equivalent to exec this command to go to the next step. Following these guidlines:
https://docs.bitnami.com/aws/infrastructure/couchdb/administration/create-ssl-certificate-couchdb/

**Comment by wohali** [comment, marked as answer]
Hi @treycalvertKM ,

First off, those are not Apache CouchDB documentation pages, those are from Bitnami. We have zero interaction with their team.

Second, `chown` is a UNIX command to change ownership of a file. You should be able to right-click on the file and change its permissions from the Windows Explorer shell.

Hope this helps!
